src/main/resources/avro/bdg.avdl

```diff
@@ -410,6 +412,19 @@ record Variant {
    is shared across all alleles in the same VCF record.
    */
   union { boolean, null } somatic = false;
+
+  /**
+  True if this site/block represents a gVCF line where no alternate allele was
+  called. This includes both gVCF modes (ERC and BP_RESOLUTION).
+   */
+  union { boolean, null } isGvcfReferenceModel = false;
```

Comment:
To keep in line with my other comments, the `is` for these flags is redundant, I'd prefer `reference` here.  Or if it is necessary to keep this specific to gVCF, `gvcfReference`.  There may also be a whitespace issue with the comment (needs 1 additional space leftpadding).